mne/epochs.py

```diff
@@ -3106,3 +3106,32 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
     _remove_meg_projs(evoked)  # remove MEG projectors, they won't apply now
     logger.info('Created Evoked dataset from %s epochs' % (count,))
     return (evoked, mapping) if return_mapping else evoked
+
+
+def segment_raw(raw, segment_length=1., verbose=True):
+    """Divide continuous raw data into equal-sized
+    consecutive chunks.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        Raw data to divide into segments.
+    segment_length : float | 1.
+        Length of each segment in seconds.
+    verbose: bool | True
+        Whether to report what is being done by printing text.
+
+    Returns
+    -------
+    epochs : instance of Epochs
+        Segmented data.
+    """
+    if not isinstance(raw, _BaseRaw):
+        raise ValueError('Input data must be an instance of Raw, got'
+                         ' %s instead.' % (type(raw)))
+    if not isinstance(segment_length, float):
+        raise ValueError('segment_length must be a float, got'
+                         ' %s instead.' % (type(segment_length)))
```

Comment:
Good point, I think I would like it, simply `raw.segment(1.)`. 
@Eric89GXL WDYT?